apps/ensrainbow/Dockerfile.data

```diff
@@ -16,30 +15,20 @@ ADD "https://bucket.ensrainbow.io/THE_GRAPH_LICENSE.txt" ./THE_GRAPH_LICENSE.txt
 RUN sha256sum -c ens_names.sql.gz.sha256sum
 
 ###################
-## Ingest Dependencies Stage — minimal deps for ingest script
+## Dependencies Stage — standard monorepo pnpm install from base
 ###################
 
 FROM base AS ingest-deps
 
-# copy pnpm files from the root directory
-COPY package.json pnpm-lock.yaml pnpm-workspace.yaml ./
+COPY pnpm-workspace.yaml ./
+COPY package.json pnpm-lock.yaml ./
+COPY packages ./packages
+COPY apps ./apps
```

Comment:
Noticed we're back to copying full directories, while previously we selected dependencies to lower the change this current docker image stage would need to be rebuilt.

Reply:
in this case it's ok because this is the data image that's built manually